### PR TITLE
docs: update HikariCP example to include configuring the datasource with a JDBC URL

### PR DESCRIPTION
**Pull request by crystall-bitquill**
### Summary

Update HikariCP example to include configuring the datasource with a JDBC URL

### Description

The existing example only uses the individual data source properties. This PR includes the option of using a URL in the example and clarifies that it is possible to use URLs.

### By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/using-the-jdbc-driver/DataSource.md [hunk 1]
```diff
@@ -79,7 +82,7 @@ To use the AWS JDBC Driver with a connection pool, you must:
    ds.addDataSourceProperty("targetDataSourceProperties", targetDataSourceProps);
    ```
 
-> **:warning:Note:** HikariCP supports either DataSource-based configuration or DriverManager-based configuration by specifying the `dataSourceClassName` or the `jdbcUrl`. When using the `AwsWrapperDataSource` you must specify the `dataSourceClassName`, therefore `HikariDataSource.setJdbcUrl` is not supported. For more information see HikariCP's [documentation](https://github.com/brettwooldridge/HikariCP#gear-configuration-knobs-baby).
+> **:warning:Note:** HikariCP supports either DataSource-based configuration or DriverManager-based configuration by specifying the `dataSourceClassName` or the `jdbcUrl`. When using the `AwsWrapperDataSource` you must specify the `dataSourceClassName`, therefore `HikariDataSource.setJdbcUrl` is not supported. However, it is possible to set the JDBC URL through the data source properties. For more information see HikariCP's [documentation](https://github.com/brettwooldridge/HikariCP#gear-configuration-knobs-baby).
```
Comment (brunos-bq) [inline docs/using-the-jdbc-driver/DataSource.md:85]: Is it possible, or should this be the way to do it?

Maybe we should mention: "The JDBC URL then needs to be defined through the data source properies, setting the `jdbcUrl` property to the wanted value. (see example)."

Comment (crystall-bitquill) [reply docs/using-the-jdbc-driver/DataSource.md:85]: It's one way of configuring the datasource, the other being setting the individual JDBC properties.

File: docs/using-the-jdbc-driver/DataSource.md [hunk 2]
```diff
@@ -64,6 +64,9 @@ To use the AWS JDBC Driver with a connection pool, you must:
    ds.addDataSourceProperty("serverName", "db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com");
    ds.addDataSourceProperty("serverPort", "5432");
    ds.addDataSourceProperty("database", "postgres");
+   
+   // The AwsWrapperDataSource can also be configured with a JDBC URL.
+   ds.addDataSourceProperty("jdbcUrl", "jdbc:aws-wrapper:postgresql://db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com:5432/postgres");
```
Comment (karenc-bq) [inline docs/using-the-jdbc-driver/DataSource.md:69]: Did we test this? IIRC you can't set jdbcUrl and servername or ds class name at the same time. Can we make sure we call that out.

Comment (karenc-bq) [reply docs/using-the-jdbc-driver/DataSource.md:69]: I see that we have some explanation in the section below, can we either link to the section below for more info or copy pasta to have the same warning in the comments?

Also does it make sense to set the protocol, servername and the jdbcUrl in the same properties? It seems really redundant.
```
   ds.addDataSourceProperty("jdbcProtocol", "jdbc:postgresql:");
   ds.addDataSourceProperty("serverName", "db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com");
  ds.addDataSourceProperty("jdbcUrl", "jdbc:aws-wrapper:postgresql://db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com:5432/postgres");
```

Comment (crystall-bitquill) [reply docs/using-the-jdbc-driver/DataSource.md:69]: You can set both the jdbcUrl and servername in the datasource, but the url takes precedence. The line to set the jdbcUrl was added to show it was possible, not necessarily that they should all be used together.

**Pull request merged**